### DISCUSSION TITLE
Add static site in /data

### DISCUSSION THREAD

**Discussion by pifou25** [NginxProxyManager/nginx-proxy-manager#1794] [Show and tell]
Hello
I use this wonderfull project as a docker container and it works fine, I love it :) really easy to set up, for me as a Nginx newbee, and efficiently, Thanks !
Today I found the way to add a static web site into /data (the shared mouted volume for docker) and added just this advanced configuration, for the proxy host sub.example.com :
```
location / {
  root /data/sub.example.com
}
```
Added common options (ssl and http/2 support works too) 'block common exploit' but disabled 'cache asset' : with this last option active, I had only the 'index.html' visible, and all other js and image resources were in error `ERR_TOO_MANY_REDIRECTS`. I could not fix the cache asset, so I just deactivate it now for this static website.

**Comment by viktak** [comment]
@pifou25 This is great idea, and would like to use it in my setup as a workaround for not being able to redirect a page to an external site.
How did you fill the other parts of the Proxy host, i.e. the `Destination` and `port`?

**Reply by itpartners-tomasz** [reply]
There is a semicolon missing at the end of the line.
The correct code should look like this:

```
location / {
  root /data/sub.example.com;
}
```

You place this in the Advanced tab of a proxy server. In forward server detail, you put anything looking correct (ex. 127.0.0.1 port 80). The data won't be used really during runtime.

I also managed to do the same trick in HOSTS >> 404 HOSTS. There you only have to provide domain name and put the code into the advanced tab.

@pifou25: THX! Good hint!

**Comment by TheSoulrester** [comment]
Sorry for this stupid question... but with this solution, I will be able to add websites like wordpress without having two different server? 

I only have one server using nginx and so I was not able to use nginx proxy manager (or was I just too stupid to set everything up correctly? nginx proxy manager would not start because the port was already in use (which is true... nginx "took" it ;) ))
